.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length=120
-ignore: E301, E302, E305, E401, F401, W503, W504, W605, E252, E402, E501, E265
+ignore: E301, E302, E305, E401, F401, W503, W504, W605, E252, E402, E501, E265, W291
```

Comment:
I don't think we should commit the sin of trailing white spaces.  >.>

Reply:
But I LIKE trailing whitespace. 😈 

This is a duplicate of https://github.com/DataBiosphere/bdcat_notebooks/pull/27#discussion_r496213696, but Brian pointed out it might not be needed on Slack. Basically, the issue is that markdown is not standardized, so *sometimes* markdown requires a double space for a newline but not always. But it looks like  <br/> works in Jupyter so that's good enough for me!